tests/security/nproc_limits.pm

```diff
@@ -17,6 +17,7 @@ sub run {
     $self->select_serial_terminal;
 
     # Check the limits.conf config file
+    script_run('test ! -f /etc/security/limits.conf && install /usr/etc/security/limits.conf -t /etc/security');
```

Comment:
Not fully correct. If `/etc/security/limts.conf` exists, it ignores `/usr/etc/security/limits.conf.d/`